generators/server/templates/src/main/docker/_app.dev.yml

```diff
@@ -72,5 +65,5 @@ services:
         <%_ } _%>
         <%_ if (applicationType == 'microservice') { _%>
         expose:
-            - "8080"
+            - "8081"
```

Comment:
Thanks for reviewing my docker PR, @jd-carroll, it's really appreciated!

About 8081 in dev profile : [application-dev.yml](https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/resources/config/_application-dev.yml#L139)
We should probably do the same thing for prod profile, which is currently 8080 : [application-prod.yml](https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/resources/config/_application-prod.yml#L136-L137)